### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by weirdan**
replaced `coffeelint` with `eslint`

### CODE REVIEW

**Comment by weirdan** [comment]
@Arcanemagus circleci broke for unrelated (I believe) reason. Now it can't find `phpcs` executable in $PATH

**Comment by Arcanemagus** [comment]
Interesting, looking into it...

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ If you would like to contribute enhancements or fixes, please do the following:
 Please note that modifications should follow these coding guidelines:
 
 - Indent is 2 spaces.
-- Code should pass `coffeelint` linter.
+- Code should pass `eslint` linter (run `npm run-script lint`).
```
Comment (Arcanemagus) [inline CONTRIBUTING.md]: A far more commonly used alias is simply `npm run scriptname`.

Comment (weirdan) [inline CONTRIBUTING.md]: Anything unambiguous would work (like `nmp ru` or `npm run-sc`). `run-script` just happens to be the actual command name, displayed in `npm help`.

Wanna get this changed to the short form?

Comment (Arcanemagus) [inline CONTRIBUTING.md]: Nah, it's fine to leave it as is. I actually didn't know that was the full command and looked it up 😛.

**Comment by Arcanemagus** [comment]
Interesting, it looks like the built in version of `composer` has changed where it places the binaries, but the CircleCI image hasn't been updated to account for this.

**Comment by Arcanemagus** [comment]
Awesome, looks like #231 fixed this up. I'll merge as soon as CI finishes.

**Comment by Arcanemagus** [comment]
Looks like AppVeyor is hitting the bug about Windows Update not being enabled. /sigh

**Comment by Arcanemagus** [comment]
Or the php package is broken _again_...

**Comment by Arcanemagus** [comment]
Looks like the Chocolatey `php` package was fixed!

**Pull request merged**